config/default_config.yaml

```diff
@@ -26,9 +26,20 @@ binance_base_currency: BTC # # This must be one of the coins listed in Binance
 binance_max_decimals_buy: 6
 binance_max_decimals_sell: 6
 
+# FIAT SETTINGS
+is_fiat: False  # TODO: Not sure if the Euro check should be here, it can be in corecito.py code just checking the <exchange>_trading_pair string
```

Comment:
The documentation here is old and can be removed.
I think adding a documentation example on this part of the file about how to setup EUR fiat on Binance will help a lot new users.

Reply:
Updated!